Sources/SwiftSyntax/Syntax.swift

```diff
@@ -23,10 +23,10 @@ public struct Syntax: SyntaxProtocol, SyntaxHashable {
   final class Info: @unchecked Sendable {
     // For root node.
     struct Root: Sendable {
-      private var arena: RetainedSyntaxArena
+      private var rawNodeArena: RetainedRawSyntaxArena
```

Comment:
You didn’t change all parameters / members from `arena` to `rawNodeArena` (eg. `RawSyntaxTokenView` still has `arena: RawSyntaxArena`). I think it would be good to have a consistent naming and I don’t see a problem with continuing to call it `arena`.

Same in some more places.